lib/tasks/import_projects_task.rb

```diff
@@ -14,23 +14,97 @@ def sync
         pp airtable_project
         puts ("-" * 20)
         proj = Project.find_or_initialize_by name: airtable_project[:project_name]
-        proj.leads = User.where(:slack_userid => airtable_project[:project_lead_slack_id]) unless airtable_project[:project_lead_slack_id].blank?
-        proj.volunteers += User.where(:slack_userid => airtable_project[:team_member_ids]) unless airtable_project[:team_member_ids].blank?
-        proj.skills = Skill.match_skills(airtable_project[:tech_stack]).tech_skills unless airtable_project[:tech_stack].blank?
+
+        if airtable_project[:project_status].blank?
+          proj.import_errors << "this project lacks a status"
+        else
+          airtable_project[:project_status].each do |status|
+            proj.status << status
+          end
+        end
+        proj.status.uniq
+
+        if airtable_project[:project_lead_slack_id].blank?
+          proj.import_errors << "this project lacks a lead"
+        else
+          airtable_project[:project_lead_slack_id].each do |slack_id|
+            lead = User.find_by(:slack_userid => slack_id)
+            if lead == nil
+              proj.import_errors << "project lead slack id #{slack_id} has no corresponding user"
+            else
+              proj.lead_ids << lead.id
+            end
+          end
+        end
+        proj.lead_ids = proj.lead_ids.uniq
+
+        airtable_project[:team_member_ids].each do |member_id|
+          volunteer = User.find_by(:slack_userid => member_id)
```

Comment:
i chose not to throw an error if there are no volunteers because i don't believe that is something that needs to be flagged - as long as there is a team lead i don't see a problem if there aren't any member ids. i'll test out the change you suggested when i'm back at my computer